config/crowbarrc

```diff
@@ -3,3 +3,4 @@ server = http://localhost:80
 username = crowbar
 password = crowbar
 anonymous = false
+apiversion = 1.0
```

Comment:
How should we do the switch to 2.0 in Cloud 7? This needs to be done somehow automatically.